.github/workflows/continuous-integration.yaml

```diff
@@ -0,0 +1,37 @@
+name: Continuous Integration
+
+on: push
+
+jobs:
+  test:
+    name: Test
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+        with:
+          fetch-depth: 1
+      - name: Setup Node.js
+        uses: actions/setup-node@v1
+        with:
+          node-version: 12
+      - name: Cache dependencies
+        uses: actions/cache@v1
+        with:
+          key:
+            npm-${{ hashFiles(format('{0}{1}', github.workspace,
+            '/package-lock.json')) }}
+          path: ~/.npm
+          restore-keys: |
+            npm-
+      - name: Cache local cache
+        uses: actions/cache@v1
+        with:
+          key: jest-${{ github.sha }}
+          path: .cache
+          restore-keys: |
+            jest-
```

Comment:
We're caching a `.cache` directory, but we call the key `jest`. I guess other caches, like ESLint, are also in this directory? Should we either change the cached directory or the name of the key?

Reply:
fixed with: https://github.com/ridedott/eslint-config/pull/319/commits/267a96360f447175bf63c69e8f81d71b5db5c652